lib/driver.js

```diff
@@ -93,7 +90,15 @@ class YouiDriver extends BaseDriver {
       if (this.driverShouldDoProxyCmd(cmd))
       {
         logger.debug(`Executing proxied WebDriver command '${cmd}'`);
-        return this.proxydriver.executeCommand(cmd, ...args);
+
+        // There are 2 CommandTimeout (YouiDriver and proxy)
+        // Only YouiDriver CommandTimeout is used; Proxy is disabled
+        // All proxy commands needs to reset the YouiDriver CommandTimeout
+        // Here we manually reset the YouiDriver CommandTimeout for commands that goe to proxy.
+        this.clearNewCommandTimeout;
+        let result = this.proxydriver.executeCommand(cmd, ...args);
+        this.startNewCommandTimeout;
```

Comment:
You should pass cmd in here, otherwise we get an string with 'undefined' logged to the server console.

this.startNewCommandTimeout(cmd);